src/App.tsx

```diff
@@ -17,18 +73,35 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelectChange={handleSelectChange}
+                query={query}
+                onInputChange={handleInputChange}
```

Comment:
The key 'onInputChange' does not follow the naming conventions for event handlers in React. Please refer to the link provided in the checklist for the recommended naming conventions.